src/Providers/providerActions.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useMemo, useState } from 'react';
+import { createActions } from 'src/extensions/actions';
+import withQueryClient from 'src/extensions/QueryClientHoc';
+import { useTranslation } from 'src/internal/i18n';
+
+import { ConfirmModal } from '@app/common/components/ConfirmModal';
+import { ProviderType } from '@app/common/constants';
+import { useDeleteProviderMutation } from '@app/queries';
```

Comment:
NOTE: ok for this PR, but don't use `@app/..` to much, it will all need to be adjusted to the sdk standarts, the idea is to remove the `pkg` dir once we have the new pages, so all the code will be using console SDK best practices.